src/main/java/org/betonquest/betonquest/BetonQuest.java

```diff
@@ -970,6 +971,7 @@ public void onEnable() {
         registerConversationIO("combined", InventoryConvIO.Combined.class);
         registerConversationIO("slowtellraw", SlowTellrawConvIO.class);
 
+        registerInterceptor("redischat", RedisChatInterceptor.class);
```

Comment:
This line shouldn't be needed at all, only registering the Integrator for the Compatibility should be enough.